internal/lsp/message_types.go

```diff
@@ -1066,6 +1066,8 @@ type OssIssueData struct {
 	ProjectName       string         `json:"projectName"`
 	DisplayTargetFile string         `json:"displayTargetFile"`
 	Details           string         `json:"details,omitempty"`
+	MatchingIssues    []OssIssueData `json:"matchingIssues"`
```

Comment:
Now the LS includes these fields in the `snyk.scan` notification

Reply:
`MatchingIssues` field is a slice of `OssIssueData`. Is it correct to say that this is a recursive data structure? Is it possible that with large data this could get complex? Is there a way to test it for deeply nested data if it make sense?